Fix IP pinning for SSO login

AntonAM opened the pull request:
We didn't get login IP when generating certificates for SSO logins, so we couldn't pin certificates. Now when creating SSO auth request (Github only in OSS repo) we save login ip in the new field, so we could use it after login is confirmed and we're generating certificates.
Most of line changes are from protobuf changes.

espadolini commented on lib/web/apiserver.go:
```diff
@@ -1401,11 +1401,18 @@ func (h *Handler) githubLoginWeb(w http.ResponseWriter, r *http.Request, p httpr
 		return client.LoginFailedRedirectURL
 	}
 
+	addr, err := utils.ParseAddr(r.RemoteAddr)
```
Isn't `ParseAddr` intended for URIs? the `RemoteAddr` of a http request is guaranteed to be a `host:port`, isn't it?

AntonAM commented:
@zmb3 yep, I'll add it 👍

AntonAM committed:
Fix IP pinning for SSO login

We didn't get login IP when generating certificates for
SSO logins, so we couldn't pin certificates.
